pkg/errors/errors.go

```diff
@@ -14,149 +14,41 @@
-	ErrorCode_UNKNOWN ErrorCode = 9000
-
-	// ErrorCode_INVALID_REQUEST 91xx
-	ErrorCode_INVALID_REQUEST           ErrorCode = 9100
-	ErrorCode_INVALID_ARGUMENT          ErrorCode = 9101
-	ErrorCode_CESQL_EXPRESSION          ErrorCode = 9102
-	ErrorCode_CEL_EXPRESSION            ErrorCode = 9103
-	ErrorCode_FILTER_ATTRIBUTE_IS_EMPTY ErrorCode = 9104
-	ErrorCode_FILTER_MULTIPLE           ErrorCode = 9105
-	ErrorCode_INVALID_HEARTBEAT_REQUEST ErrorCode = 9106
-	ErrorCode_JSON_PARSE                ErrorCode = 9107
-	ErrorCode_TRANSFORM_INPUT_PARSE     ErrorCode = 9108
-	ErrorCode_CORRUPTED_EVENT           ErrorCode = 9109
-
-	// ErrorCode_SERVICE_NOT_RUNNING 92xx
-	ErrorCode_SERVICE_NOT_RUNNING           ErrorCode = 9200
-	ErrorCode_SEGMENT_SERVER_HAS_BEEN_ADDED ErrorCode = 9201
-	ErrorCode_SERVICE_STATE_ERROR           ErrorCode = 9202
-	ErrorCode_WORKER_NOT_RUNNING            ErrorCode = 9203
-
-	// ErrorCode_RESOURCE_EXIST 93xx
-	ErrorCode_RESOURCE_EXIST ErrorCode = 9300
-
-	// ErrorCode_RESOURCE_NOT_FOUND 94xx
-	ErrorCode_RESOURCE_NOT_FOUND ErrorCode = 9400
-	ErrorCode_EVENTBUS_NOT_FOUND ErrorCode = 9401
-	ErrorCode_EVENTLOG_NOT_FOUND ErrorCode = 9402
-	ErrorCode_SEGMENT_NOT_FOUND  ErrorCode = 9403
-	ErrorCode_BLOCK_NOT_FOUND    ErrorCode = 9404
-	ErrorCode_VOLUME_NOT_FOUND   ErrorCode = 9405
-
-	// ErrorCode_INTERNAL 95xx
-	ErrorCode_INTERNAL               ErrorCode = 9500
-	ErrorCode_TRIGGER_WORKER         ErrorCode = 9501
-	ErrorCode_INVALID_SEGMENT        ErrorCode = 9502
-	ErrorCode_INVAILD_HEAETBEAT      ErrorCode = 9503
-	ErrorCode_VOLUME_NO_SERVER       ErrorCode = 9504
-	ErrorCode_JSON_MARSHAL           ErrorCode = 9505
-	ErrorCode_JSON_UNMARSHAL         ErrorCode = 9506
-	ErrorCode_AES_ENCRYPT            ErrorCode = 9507
-	ErrorCode_AES_DECRYPT            ErrorCode = 9508
-	ErrorCode_UNMARSHAL              ErrorCode = 9509
-	ErrorCode_LAMBDA_INVOKE          ErrorCode = 9510
-	ErrorCode_LAMBDA_INVOKE_RESPONSE ErrorCode = 9511
-
-	// ErrorCode_SEGMENT_FULL 96xx
-	ErrorCode_SEGMENT_FULL            ErrorCode = 9600
-	ErrorCode_SEGMENT_NO_ENOUGH_SPACE ErrorCode = 9601
-	ErrorCode_OFFSET_UNDERFLOW        ErrorCode = 9602
-	ErrorCode_OFFSET_OVERFLOW         ErrorCode = 9603
-	ErrorCode_OFFSET_ON_END           ErrorCode = 9604
-	ErrorCode_BLOCK_NOT_SUPPORTED     ErrorCode = 9605
-	ErrorCode_NOT_WRITABLE            ErrorCode = 9606
-	ErrorCode_NOT_READABLE            ErrorCode = 9607
-	ErrorCode_TRY_AGAIN               ErrorCode = 9608
-	ErrorCode_NO_ENDPOINT             ErrorCode = 9609
-	ErrorCode_CLOSED                  ErrorCode = 9610
-
-	// ErrorCode_NOT_LEADER 97xx
-	ErrorCode_NOT_LEADER           ErrorCode = 9700
-	ErrorCode_NO_CONTROLLER_LEADER ErrorCode = 9701
-	ErrorCode_NOT_RAFT_LEADER      ErrorCode = 9702
-
-	// ErrorCode_RESERVE 98xx
-
-	// ErrorCode_OTHERS 99xx
-	ErrorCode_RESOURCE_EXHAUSTED  ErrorCode = 9901
-	ErrorCode_RESOURCE_CAN_NOT_OP ErrorCode = 9902
+import (
+	errpb "github.com/linkall-labs/vanus/proto/pkg/errors"
```

Comment:
Changing error in this PR is not a good idea.

Reply:
In order to reuse error codes in the interface code, I admit that this is not a good idea.